pages.html/pg1.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html>
+
+    <head>
+            <title>MY RESUME</title>
+        <header>
+            <a href="pg2.html">Contact</a>
+            <h1>Denae Brown</h1>
+            <img src="C:\Users\denae\Portfolio-Lab\page.1.HTML\Me.JPG" alt="photo of me"/>
```

Comment:
Image did not load correctly, you used an absolute file path. If you used a relative one like `Me.JPG` it would have loaded on mine as well as yours (-1)